src/styles/index.scss

```diff
@@ -1,3 +1,17 @@
-body {
+@import '/src/styles/utilis/mixins';
+@import '/src/styles/utilis/variables';
+@import '/src/styles/blocks/card';
+@import '/src/styles/blocks/stars';
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  bottom: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
```

Comment:
Consider moving general page styles into a separate file to keep your main file clean and clear